src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java

```diff
@@ -133,7 +130,49 @@ public void deleteRevisionEntry(String commitHash) throws IllegalArgumentExcepti
    * Used to separate the usage from the actual Spring Datastore implementation.
    * Returns null when nothing was found.
    */
-  private BuildInfo getRevisionEntry(String commitHash) {
+  public BuildInfo getRevisionEntry(String commitHash) {
     return storage.findById(commitHash, BuildInfo.class);
   }
+
+  /**
+   * Queries all "revision" type entries, and deletes all which are older than a specified
+   * amount of time. During this operation, the GCDataRepository can be queried, but
```

Comment:
```suggestion
   * amount of time. During this operation, this repository can be queried, but
```
(GCDataRepository is the obsolete name of this class)